app/assets/stylesheets/content/_forms.sass

```diff
@@ -57,6 +57,8 @@ $form--field-types: (text-field, text-area, select, check-box, radio-button, ran
     padding: 30px 20px
     background-color: $content-form-bg-color
     border: $content-form-border
+  &.-higher
+    min-height: 31em
```

Comment:
as i understand this sets a minimum height on the form which happens to be one way of revealing the tooltips.